### ISSUE TITLE
Asking about the training and valid set.

### ISSUE THREAD

**Issue by wcycqjy** [starrytong/SCNet#29]
Thanks for your work. I see you mention there are "train" and "valid" in the data folder. But I don't find there is any official valid set for MUSDB18HQ. Do you mean using the test set as the valid one?

**Comment by starrytong** [comment]
If necessary, you can select several songs from the training set as the validation set. I used some songs from the MoisesDB as the validation set.

**Comment by wcycqjy** [comment]
Can you tell me which songs did you use in MoisesDB?

**Comment by starrytong** [comment]
Sorry, I forgot the specific songs. You can randomly select 10-20 songs. I believe it won't significantly affect the final training results.
